.github/workflows/push.yml

```diff
@@ -0,0 +1,23 @@
+name: "Push Checks"
+on:
+  push:
+    paths:
+      - "terraform/**"
+defaults:
+  run:
+    working-directory: terraform/deploy/
+
+jobs:
+  terraform:
+    name: "Terraform"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: 0.12.19
+
+      - name: "Terraform Format"
+        id: fmt
+        run: terraform fmt
+        continue-on-error: true
```

Comment:
is the plan to add `apply` later? If so, maybe we could have the job put in now but just have it do a `plan` for now, just so we can see everything is in working order

Reply:
yes there will be a job for merging to master that runs the apply.  We're not there yet.